src/lib/utils/application-commands/computeDifferences.ts

```diff
@@ -644,6 +644,61 @@ function* reportOptionDifferences({
 			}
 		}
 	}
+
+	if (hasMinMaxLengthSupport(option)) {
```

Comment:
What I don't get is how the computeDifference logs will differ from the object-hash logs and why it is worth keeping compute-difference if we have object-hash to compare as well

Reply:
Read from here https://canary.discord.com/channels/737141877803057244/737142774738190377/1001514680403103875 down

Reply:
Yes but object-hash still prints key/value pairs so wouldn't that also suffice for Kaname's needs?

Reply:
It doesn't, it makes a SHA1 hash of all values, and it wouldn't print the differences, just that they are different